From 34a6331b1f22955988d0d1c955be54e50e59c08b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <loic.minier@linaro.org>
Date: Thu, 3 Feb 2011 12:35:38 +0100
Subject: [PATCH] EfikaMX: switch to MACH_TYPE_MX51_EFIKAMX

Upstream linux moved from MACH_TYPE_MX51_LANGE51 to
MACH_TYPE_MX51_EFIKAMX.
---
 board/efikamx/efikamx.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/board/efikamx/efikamx.c b/board/efikamx/efikamx.c
index 4efbeaf..f735260 100644
--- a/board/efikamx/efikamx.c
+++ b/board/efikamx/efikamx.c
@@ -621,7 +621,7 @@ int board_early_init_f(void)
 
 int board_init(void)
 {
-	gd->bd->bi_arch_number = MACH_TYPE_MX51_LANGE51;
+	gd->bd->bi_arch_number = MACH_TYPE_MX51_EFIKAMX;
 	gd->bd->bi_boot_params = PHYS_SDRAM_1 + 0x100;
 
 	return 0;
-- 
1.7.2.3

